src/DataViews/include/DataViews/AppInterface.h

```diff
@@ -51,7 +51,7 @@ class AppInterface {
   virtual void SetHighlightedFunctionId(uint64_t highlighted_function_id) = 0;
   virtual void SetVisibleFunctionIds(absl::flat_hash_set<uint64_t> visible_functions) = 0;
   virtual void DeselectTimer() = 0;
-  [[nodiscard]] virtual bool IsCapturing() const = 0;
+  [[nodiscard]] virtual bool IsDataStreamingActive() const = 0;
```

Comment:
This change seems to be rather unrelated. Is there a reason for renaming the function?

Reply:
Ah, is it about having it in both interfaces? Why not the same name?

Reply:
Yes, it's about having it in both interfaces. I was assuming the same name would cause issues, I didn't even consider trying it to be honest...

Reply:
Woo! I also didn't know they can have the same! Good to know that XD

Reply:
TIL that this works. That's better, thanks.